internal/cmd/config/set/set.go

```diff
@@ -124,6 +126,8 @@ Use "{{.CommandPath}} [command] --help" for more information about a command.{{e
 
 func configureFlags(cmd *cobra.Command) {
 	cmd.Flags().String(sessionTimeLimitFlag, "", "Maximum time before authentication is required again. After this time, you will be prompted to login again to execute commands that require authentication. Can't be larger than 24h. Requires authentication after being set to take effect. Examples: 3h, 5h30m40s (BETA: currently values greater than 2h have no effect)")
+	cmd.Flags().String(identityProviderCustomEndpointFlag, "", "Identity Provider base URL, used for user authentication")
```

Comment:
I think that here we could say that the IDP needs to be OICD compliant (with a relevant link to the protocol documentation)

Reply:
As discussed, will only allow STACKIT domains for now